packages/slate-react/src/components/content.js

```diff
@@ -468,10 +466,7 @@ class Content extends React.Component {
       return this.renderNode(child, isSelected, childrenDecorations[i])
     })
 
-    const handlers = EVENT_HANDLERS.reduce((obj, handler) => {
-      obj[handler] = this[handler]
-      return obj
-    }, {})
+    const handlers = this.getHandlers()
```

Comment:
im not sure why the initial implementation is the way it is, but this is actually creating new behavior (new function was being created before on every render, now its not).

are we sure this doesn't subtly change anything?